Docs/UPDATING.md

```diff
@@ -52,8 +54,11 @@ To set up `BazelXCBBuildService` for debugging:
 	cp ~/Library/Developer/Xcode/DerivedData/$FOLDER/Build/Products/Debug/BazelXCBBuildService \
 	/Applications/Xcode-<main-version>.app/Contents/SharedFrameworks/XCBuild.framework/PlugIns/XCBBuildService.bundle/Contents/MacOS/
 	```
-1. In the "debugging" Xcode Go to "Debug" > "Attach to process by PID or Name" and type in "BazelXCBBuildService" if it's not already pre-filled.
+1. Prepare the debugger to attach to the service:
+	1. Open the scheme for this build, and navigate to the "Run" section on the left-hand sidebar
+	1. In the "Info" tab, for the "Launch" option, select "Wait for the executable to be launched"
 1. Set some breakpoints if desired
```

Comment:
I would make sure all of these are sentences too, with periods at the end.

Reply:
Whoops! Good catch.